### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by chris-brugal**
Started eslint and prettier. Using husky so that prettier runs before commit occurs.

### CODE REVIEW

File: frontend/src/components/CoursePage.js [hunk 1]
```diff
@@ -1,360 +1,377 @@
-import React from 'react'
-import { Button, Stack, CardHeader, CardContent, Rating, Card, Typography, Grid, TextField, Paper, Box } from '@mui/material'
+import React from "react";
```
Comment (cole-kitroser) [inline frontend/src/components/CoursePage.js:1]: This change feels unnecessary.

**Review by cole-kitroser — APPROVED** [review]
Outside of one comment I had on CoursePage, everything looks good.

File: frontend/src/components/CoursePage.js (hunk 1 above)
Comment (WillBAnders) [reply frontend/src/components/CoursePage.js:1]: Everything here is styling changes, not sure if there's something in particular you're pointing to.

**Review by WillBAnders — APPROVED** [review]

**Comment by WillBAnders** [comment]
Reverted for now, there are a few issues with the newline character being applied differently based on the operating system and some linting checks resulting in compiler errors with `npm run start`.

**Pull request merged**